### ISSUE TITLE
`get_element_bt_name("basement")` -> ValueError: Element with name "basement" not found in the structural frame.

### ISSUE THREAD

**Issue by JoostGevaert-Arup** [cgre-aachen/gempy#870]
**Describe the bug**
I saw [Florian Wellman's comment](https://www.linkedin.com/feed/update/urn:li:ugcPost:7175323303488737280?commentUrn=urn%3Ali%3Acomment%3A%28ugcPost%3A7175323303488737280%2C7175449402365345792%29&dashCommentUrn=urn%3Ali%3Afsd_comment%3A%287175449402365345792%2Curn%3Ali%3AugcPost%3A7175323303488737280%29) on someone's LinkedIn post, saying that GemPy v3 is underway, and that there is a [Google Colab](https://colab.research.google.com/drive/1JhXMJrGrJeXJK2MOB-btrEb_eyA8hk3o).

When trying out the Google Colab, after Step 5, I tried to get some structural elements by name:
```Python
print(geo_model.structural_frame.elements_names)
print(geo_model.structural_frame.get_element_by_name("basement"))
```
But then got the error:
```
['surface2', 'surface1', 'basement']
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
[<ipython-input-35-1168d3e7cf55>](https://localhost:8080/#) in <cell line: 2>()
      1 print(geo_model.structural_frame.elements_names)
----> 2 print(geo_model.structural_frame.get_element_by_name("basement"))

[/usr/local/lib/python3.10/dist-packages/gempy/core/data/structural_frame.py](https://localhost:8080/#) in get_element_by_name(self, element_name)
     35         element = next(valid_elements, None)
     36         if element is None:
---> 37             raise ValueError(f"Element with name {element_name} not found in the structural frame.")
     38         return element
     39 

ValueError: Element with name basement not found in the structural frame.
```

Which I find curious, because `"basement"` is listed when running `print(geo_model.structural_frame.elements_names)`

**To Reproduce**
Run [GemPy v3 Google Colab](https://colab.research.google.com/drive/1JhXMJrGrJeXJK2MOB-btrEb_eyA8hk3o)
...

**Expected behavior**
I expected `geo_model.structural_frame.get_element_by_name("basement")` to return the `"basement" <class 'gempy.core.data.structural_element.StructuralElement'>`.

**Screenshots**
![image](https://github.com/cgre-aachen/gempy/assets/75024225/3b8cbde9-c6d7-4d48-ab8a-a541e1a22630)

**Desktop (please complete the following information):**
N/A

**Comment by JoostGevaert-Arup** [comment]
I can obtain `"surface1"` and `"surface2"`, I guess because they're in a `<class 'gempy.core.data.structural_group.StructuralGroup'`?
However, I do find it somewhat confusing that `"basement"` is listed when I do `print(geo_model.structural_frame.elements_names)`, but then cannot obtain it.

**Comment by flohorovicic** [comment]
Hi @JoostGevaert-Arup,

Happy to see that you are giving the online GemPy version a try! 

The reason that you can not get more information about the basement is that it is - by default- the base of the stratigraphic pile - i.e.: everything below the first structural element that you define ("surface1", in this case). As such, it does not have any data associated with it (no interface points, orientations, etc.).

Can you explain a bit more what exactly you try to achieve and what you need it for? Maybe we can solve the issue from there?

**Comment by JoostGevaert-Arup** [comment]
Hi Florian,

Great to hear from you!
I'm a big fan of open-source software and have always been looking for ways to use GemPy in my work in geotechnical engineering, so I was very happy to see it'll now become easier to get my colleagues introduced to GemPy with the Google Colab :)

I was not really trying to achieve anything specific. I was just exploring GemPy in that Colab and then stumbled upon the error mentioned above.

Rather than returning a ValueError when trying to retrieve the "basement", it might make more sense to return a message similar to what you just explained above?

**Comment by flohorovicic** [comment]
Yes, good idea! I'll add this as a development note. Just as an information: we are also working on more notebooks for Google Colab with additional examples and use cases. Hope this helps you and your colleagues in getting forward with GemPy!
